src/ontology/mondo-edit.obo

```diff
@@ -155366,7 +155366,6 @@ xref: Orphanet:2036 {source="MONDO:equivalentTo", source="GARD:0000159", source=
 xref: SCTID:721888002 {source="MONDO:equivalentTo"}
 xref: UMLS:C1867020 {source="Orphanet:2036", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:2036/e", source="OMIM:181270"}
 is_a: MONDO:0015161 {source="Orphanet:2036"} ! multiple congenital anomalies/dysmorphic syndrome without intellectual disability
-is_a: MONDO:0015853 {source="Orphanet:2036"} ! deficient breast volume or number
```

Comment:
I think it is okay that 'scalp-ear-nipple syndrome' left the breast disorder branch